FlowCrypt/Functionality/Mail Provider/EmptyInboxProvider/Gmail+EmptyInbox.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Gmail+EmptyInbox.swift
+//  FlowCrypt
+//
+//  Created by Ioan Moldovan on 6/28/22
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import FlowCryptCommon
+import GoogleAPIClientForREST_Gmail
+
+extension GmailService: EmptyInboxProvider {
```

Comment:
EmptyFolderProvider maybe. But is that really needed? Can't use MessageOperations.batchDeleteMessages directly? I'd like to minimize the amount of classes that just have one method with a few lines - no value add unless there is a good reason. The `emptyFolder` code could be in `MessageOperations` maybe.